### PR TITLE
Added a task Atbash Cipher

### PR DESCRIPTION
**Pull request by annaoliinyk**
https://exercism.org/tracks/python/exercises/atbash-cipher

Instructions
Create an implementation of the atbash cipher, an ancient encryption system created in the Middle East.

The Atbash cipher is a simple substitution cipher that relies on transposing all the letters in the alphabet such that the resulting alphabet is backwards. The first letter is replaced with the last letter, the second with the second-last, and so on.

An Atbash cipher for the Latin alphabet would be as follows:

Plain:  abcdefghijklmnopqrstuvwxyz
Cipher: zyxwvutsrqponmlkjihgfedcba
It is a very weak cipher because it only has one possible key, and it is a simple mono-alphabetic substitution cipher. However, this may not have been an issue in the cipher's time.

Ciphertext is written out in groups of fixed length, the traditional group size being 5 letters, leaving numbers unchanged, and punctuation is excluded. This is to make it harder to guess things based on word boundaries.

Examples
- Encoding test gives gvhg
- Encoding x123 yes gives c123b vh
- Decoding gvhg gives test
- Decoding gsvjf rxpyi ldmul cqfnk hlevi gsvoz abwlt gives thequickbrownfoxjumpsoverthelazydog

### CODE REVIEW

File: atbash_cipher/atbash_cipher_task.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Task from https://exercism.org/tracks/python/exercises/atbash-cipher
+
+def encode(plain_text):
+    plain_str = "abcdefghijklm0123456789nopqrstuvwxyz"
+    cipher_str = "zyxwvutsrqpon0123456789mlkjihgfedcba"
```
Comment (AnaKovaGH) [inline atbash_cipher/atbash_cipher_task.py]: Make it separate constant outside the function

File: atbash_cipher/atbash_cipher_task.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        if symbol in plain_str:
+            index = plain_str.index(symbol)
+            result += cipher_str[index]
+            letters_counter += 1
+    return result
+
+
+def decode(ciphered_text):
+    plain_str = "abcdefghijklm0123456789nopqrstuvwxyz"
+    cipher_str = "zyxwvutsrqpon0123456789mlkjihgfedcba"
```
Comment (AnaKovaGH) [inline atbash_cipher/atbash_cipher_task.py]: Same here

File: atbash_cipher/atbash_cipher_task.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+def encode(plain_text):
+    plain_str = "abcdefghijklm0123456789nopqrstuvwxyz"
+    cipher_str = "zyxwvutsrqpon0123456789mlkjihgfedcba"
+    result = ""
+    letters_counter = 0
+    for symbol in plain_text.lower():
+        if letters_counter == 5 and symbol in plain_str:
+            result += " "
+            letters_counter = 0
+        if symbol in plain_str:
```
Comment (AnaKovaGH) [inline atbash_cipher/atbash_cipher_task.py]: Please change these if statement as it has the same condition as in previous one

File: atbash_cipher/atbash_cipher_task.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+
+def decode(ciphered_text):
+    plain_str = "abcdefghijklm0123456789nopqrstuvwxyz"
+    cipher_str = "zyxwvutsrqpon0123456789mlkjihgfedcba"
+    result = ""
+    for symbol in ciphered_text.lower():
+        if symbol in cipher_str:
+            index = cipher_str.index(symbol)
+            result += plain_str[index]
```
Comment (AnaKovaGH) [inline atbash_cipher/atbash_cipher_task.py]: Lines 25 and 26 looks quite similar to 13 and 14. Maybe there is a reason why it should be in separate function?

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change what was commented above.

**Comment by annaoliinyk** [comment]
The issues were fixed, simplified program readability, added main() method and logging

**Comment by annaoliinyk** [comment]
made constants uppercase, changed logging from debug to info

**Review by AnaKovaGH — APPROVED** [review]

**Pull request merged**